### PR TITLE
Prep for new release

### PR DESCRIPTION
**Pull request by mberacochea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +81,7 @@ nextflow run ebi-metagenomics/shallowmapping \
    --decont_reference_paths <DECONT_REFS_PATH/reference_genomes>
 ```
 
-At the moment, the biome selection is limited to the precomputed databases for [chicken-gut-v1-0-1](https://www.ebi.ac.uk/metagenomics/genome-catalogues/chicken-gut-v1-0-1) and [mouse-gut-v1-0](https://www.ebi.ac.uk/metagenomics/genome-catalogues/mouse-gut-v1-0). Other databases can be built for any of the [`MGnify genome catalogues`](https://www.ebi.ac.uk/metagenomics/browse/genomes) upon request by opening an issue in this repository (they will be built on a best-effort basis).
+At the moment, the biome selection is limited to the precomputed databases for [chicken-gut-v1-0-1](https://www.ebi.ac.uk/metagenomics/genome-catalogues/chicken-gut-v1-0-1), [mouse-gut-v1-0](https://www.ebi.ac.uk/metagenomics/genome-catalogues/mouse-gut-v1-0), and [human-gut-v2-0-2](https://www.ebi.ac.uk/metagenomics/genome-catalogues/human-gut-v2-0-2). Other databases can be built for any of the [`MGnify genome catalogues`](https://www.ebi.ac.uk/metagenomics/browse/genomes) upon request by opening an issue in this repository (they will be built on a best-effort basis).
```
Comment (KateSakharova) [inline README.md]: This one repeats info in line 8, right? I guess we do not need those line here

File: bin/panaroo_inputs_builder.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+def metadata_parser(catalogue_metadata):
+    reps_clusters = {}
+    with open(catalogue_metadata, "r") as input_file:
+        next(input_file)
+        for line in input_file:
+            l_line = line.rstrip().split("\t")
+            clstr_member = l_line[0]
+            rep_genome = l_line[13]
+            if rep_genome in reps_clusters:
```
Comment (KateSakharova) [inline bin/panaroo_inputs_builder.py]: setdefault? ;)

File: bin/panaroo_inputs_builder_custom.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+def metadata_parser( drep_clstrs, derep_genomes ):
+    clusters = {}
+    with open(drep_clstrs, "r") as input_file:
+        next(input_file)
+        for line in input_file:
+            l_line = line.rstrip().split(",")
+            clstr_member = l_line[0].replace('.fa','')
+            clstr_num = 'clstr_'+l_line[5]
+            if clstr_num in clusters:
```
Comment (KateSakharova) [inline bin/panaroo_inputs_builder_custom.py:26]: and here setdefault too

File: bin/pangenomeDB_builder_custom.py [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+                integrated_dict[rep_gene] = annot_data
+            for acc_gene, acc_annot_data in acc_gff_dict.items():
+                if len(acc_annot_data) == 7:
+                    integrated_dict[acc_gene] = acc_annot_data
+
+            '''
+
+            output_writer(acc_gff_dict, rep, clstr_size, core_names)
+            
+
```
Comment (KateSakharova) [inline bin/pangenomeDB_builder_custom.py]: extra lines

File: bin/pangenomeDB_builder_custom.py [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+                # Parsing the gff file of relevant members including the rep genome
+                mem_gff_loc = annot_loc + member + ".gff"
+                acc_gff_dict = acc_gff_parser(mem_gff_loc, relevant_genes, acc_gff_dict)
+
+                # Parsing the eggNOG annotation of relevant genes
+                eggnog_annot = annot_loc + member + "_eggnog.emapper.annotations"
+                if os.path.exists(eggnog_annot):
+                    acc_gff_dict = acc_eggnog_parser(eggnog_annot, relevant_genes, acc_gff_dict, pfam_desc)
+
+
```
Comment (KateSakharova) [inline bin/pangenomeDB_builder_custom.py]: extra lines

File: bin/pangenomeDB_builder_custom.py [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+                acc_gff_dict[gene].append(cazy_annot[gene])
+            else:
+                acc_gff_dict[gene].append("-")
+            if gene in pfam_annot:
+                acc_gff_dict[gene].append(pfam_annot[gene])
+            else:
+                acc_gff_dict[gene].append("-")
+
+    return acc_gff_dict
+
```
Comment (KateSakharova) [inline bin/pangenomeDB_builder_custom.py:322]: extra lines

**Review by KateSakharova — APPROVED** [review]
minor fixes required

File: bin/panaroo_inputs_builder.py (hunk 2 above)
Comment (mberacochea) [reply bin/panaroo_inputs_builder.py]: yeah.. we need to refactor some of the scripts to make those kind of small improvements. But.. later

File: README.md (hunk 1 above)
Comment (mberacochea) [reply README.md]: indeed!

File: bin/panaroo_inputs_builder_custom.py (hunk 3 above)
Comment (mberacochea) [reply bin/panaroo_inputs_builder_custom.py:26]: Yep, same as the previuos one.

**Pull request merged**